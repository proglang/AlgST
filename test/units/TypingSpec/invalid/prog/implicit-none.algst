-- Compare with
--
--   * valid/prog/implicit-top-level.aglst
--   * valid/prog/implicit-local.algst
--   * valid/prog/implicit-higher-order-call.algst
--
-- which all share the same structure but provide the requested implicit `Int`
-- on different levels.

calc : Int -> Int

f : Int
f = calc (implicit[Int] + 10)
