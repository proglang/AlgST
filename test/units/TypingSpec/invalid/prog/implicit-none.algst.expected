
13:20: error:
	 No implicit available of type
	     Int